src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DriverComponent.scala

```diff
@@ -17,6 +19,7 @@ trait DriverComponent {
 
   // needed by MySQL but not actually used; we will always overwrite
   val defaultTimeStamp = Timestamp.valueOf("2001-01-01 01:01:01.0")
+  val batchSize = ConfigFactory.parseFile(new File("/etc/rawls.conf")).getConfig("slick").getInt("batchSize")
```

Comment:
no, val batchSize should be abstract here. DataAccessComponent is the concrete class that is constructed in Boot, pass in the batch size there